projects/js-packages/publicize-components/src/components/services/custom-inputs.tsx

```diff
@@ -83,12 +109,23 @@ export function CustomInputs( { service }: CustomInputsProps ) {
 						aria-label={ __( 'Bluesky handle', 'jetpack-publicize-components' ) }
 						aria-describedby={ `${ id }-handle-description` }
 						placeholder={ 'username.bsky.social' }
+						onChange={ handleBskyHandleChange }
+						className={ handleError ? styles.error : undefined }
 					/>
-					<p className="description" id={ `${ id }-handle-description` }>
-						{ __(
-							'You can find the handle in your Bluesky profile.',
-							'jetpack-publicize-components'
-						) }
+					<p
+						className={ clsx( 'description', handleError && styles[ 'error-text' ] ) }
+						id={ `${ id }-handle-description` }
+					>
+						{ handleError ||
+							createInterpolateElement(
+								__(
+									'You can find the handle in your Bluesky profile. This will be either <strong>username.bsky.social</strong> or just the domain name if you are using a custom domain.',
+									'jetpack-publicize-components'
+								),
+								{
+									strong: <strong />,
+								}
+							) }
 					</p>
```

Comment:
Here I think it's better to retain the existing string to re-use its translation and separate the other one be `&nbsp;`. Also, it's better to have cleaner i18n strings without HTML because it's annoying to handle them in RTL languages.

```suggestion
					<p
						className={ clsx( 'description', handleError && styles[ 'error-text' ] ) }
						id={ `${ id }-handle-description` }
					>
						{ handleError || (
							<>
								{ __(
									'You can find the handle in your Bluesky profile.',
									'jetpack-publicize-components'
								) }
								&nbsp;
								{ createInterpolateElement(
									sprintf(
										/* translators: %s is the bluesky handle suffix like .bsky.social */
										__(
											'This can either be %s or just the domain name if you are using a custom domain.',
											'jetpack-publicize-components'
										),
										'<strong>username.bsky.social</strong>'
									),
									{
										strong: <strong />,
									}
								) }
							</>
						) }
					</p>
```